animation_nodes/nodes/mesh/bmesh_invert_normals.py

```diff
@@ -0,0 +1,23 @@
+import bpy
+from bpy.props import *
+from ... events import executionCodeChanged
+from ... base_types import AnimationNode
+
+class BMeshInvertNormalsNode(bpy.types.Node, AnimationNode):
+    bl_idname = "an_BMeshInvertNormalsNode"
+    bl_label = "BMesh Invert Normals"
+
+    flipMultires: BoolProperty(name = "Flip Multires", update = executionCodeChanged)
```

Comment:
The execution code no longer changes, so `executionCodeChanged` shouldn't be used here. Instead, you should use `propertyChanged`.